code/modules/computer3/NTOS.dm

```diff
@@ -1,239 +0,0 @@
-/*
-	The Big Bad NT Operating System
-*/
-
-/datum/file/program/ntos
-	name = "Nanotrasen Operating System"
-	extension = "prog"
-	active_state = "ntos"
-	var/obj/item/part/computer/storage/current // the drive being viewed, null for desktop/computer
-	var/fileop = "runfile"
-
-/*
-	Generate a basic list of files in the selected scope
-*/
-
-/datum/file/program/ntos/proc/list_files()
-	if(!computer || !current) return null
-	return current.files
-
-
-/datum/file/program/ntos/proc/filegrid(var/list/filelist)
```

Comment:
Hmm, this UI component is one of the things I'd like to bring over to the modular computers system